synapse/storage/schema/main/delta/74/90COMMENTS_destinations.sql.postgres

```diff
@@ -21,16 +21,21 @@ COMMENT ON TABLE destinations IS
 COMMENT ON COLUMN destinations.destination IS 'server name of remote homeserver in question';
 
 COMMENT ON COLUMN destinations.last_successful_stream_ordering IS
-    'Stream ordering of the most recently successfully sent PDU to this server. See `destination_rooms` for more information about catch-up.';
+$$Stream ordering of the most recently successfully sent PDU to this server, sent through normal send (not e.g. backfill).
+In Catch-Up Mode, the original PDU persisted by us is represented here, even if we sent a later forward extremity in its stead.
+See `destination_rooms` for more information about catch-up.$$;
 
 COMMENT ON COLUMN destinations.retry_last_ts IS
-    'The last time we tried and failed to reach the remote server, in ms.';
+$$The last time we tried and failed to reach the remote server, in ms.
+This field is reset to `0` when we succeed in connecting again.$$;
 
 COMMENT ON COLUMN destinations.retry_interval IS
-    'How long since the last time we tried to reach the remote server before trying again, in ms.';
+$$How long, in milliseconds, to wait since the last time we tried to reach the remote server before trying again.
+This field is reset to `0` when we succeed in connecting again.$$;
 
 COMMENT ON COLUMN destinations.failure_ts IS
-    'The first time we tried and failed to reach the remote server, in ms.';
+$$The first time we tried and failed to reach the remote server, in ms.
+This field is reset to `NULL` when we succeed in connecting again.$$;
```

Comment:
Slightly infuritating that it's 0 for some of them but NULL for others!

Reply:
Yes, maybe there's a reason but I don't know. /shrug